pyrogram/storage/file_storage.py

```diff
@@ -1,5 +1,5 @@
 #  Pyrogram - Telegram MTProto API Client Library for Python
-#  Copyright (C) 2017-present Dan <https://github.com/delivrance>
+#  Copyright (C) 2017-2021 Dan <https://github.com/delivrance>
```

Comment:
2021?

Reply:
I need to merge the master branch to update the changes

Reply:
i see